components/Chatbar/components/ChatbarSettings.tsx

```diff
@@ -32,29 +33,25 @@ export const ChatbarSettings = () => {
   } = useContext(HomeContext);
 
   const {
-    handleClearConversations,
-    handleImportConversations,
-    handleExportData,
     handleApiKeyChange,
   } = useContext(ChatbarContext);
 
-  return (
-    <div className="flex flex-col items-center space-y-1 border-t border-white/20 pt-1 text-sm">
-      {conversations.length > 0 ? (
-        <ClearConversations onClearConversations={handleClearConversations} />
-      ) : null}
-
-      <Import onImport={handleImportConversations} />
-
-      <SidebarButton
-        text={t('Export data')}
-        icon={<IconFileExport size={18} />}
-        onClick={() => handleExportData()}
-      />
+  const getInitials = (name: string) => {
```

Comment:
We could have people w/more than 2 initials as found by this code, so we should check the display for that at some point.